e2e/cypress/integration/multi_team_and_dm/max_gm_members_spec.js

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @multi_team_and_dm
+
+describe('Multi-user group messages', () => {
+    let testUser;
+    let testTeam;
+    before(() => {
+        // # Create a new team
+        cy.apiInitSetup().then(({team, user}) => {
+            testUser = user;
+            testTeam = team;
+
+            // # Add 10 users to the team
+            Cypress._.times(10, () => createUserAndAddToTeam(testTeam));
+        });
+    });
+
+    it('MM-T463 Should not be able to create a group message with more than 8 users', () => {
+        // # Login as test user
+        cy.apiLogin(testUser);
+
+        // # Go to town-square channel
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+
+        // # Open the 'Direct messages' dialog to create a new direct message
+        cy.get('#addDirectChannel').click();
+
+        // # Add the maximum amount of users to a group message (7)
+        addUsersToGMViaModal(7);
+
+        // * Check that the user count in the group message modal equals 7
+        cy.get('.react-select__multi-value').should('have.length', 7);
+
+        expectCannotAddUsersMessage(0);
+
+        // # Try to add one more user
+        addUsersToGMViaModal(1);
+
+        // * Check that the list of users in the multi-select is still 7
+        cy.get('.react-select__multi-value').should('have.length', 7);
+
+        // # Click on "Go" in the group message's dialog to begin the conversation
+        cy.get('#saveItems').click();
+
+        // * Check that the number of users in the group message is 8
+        cy.get('#channelMemberCountText').
+            should('be.visible').
+            and('have.text', '8');
+
+        // * Check that the direct message text input's placeholder is truncated if the user's list is too long
+        cy.get('[data-testid="post_textbox_placeholder"]').
```

Comment:
I'm good as is. Can simply be
```suggestion
        cy.findByTestId('post_textbox_placeholder').
```